### PR TITLE
installation: Add system requirements page

### PR DESCRIPTION
**Pull request by TraceyC77**
## Description

Add a page to outline system requirements
Link from the Quick Start page

Fixes /getsolus/help-center-docs/issues/506

### CODE REVIEW

**Comment by TraceyC77** [comment]
Related PR for the Download page on the main site
https://github.com/getsolus/solus-site.github.io/pull/27

File: docs/user/quick-start/installation/system-requirements.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+| **Disk space free**       | 10GB / 9.3GiB                   | 25GB / 23.3GiB                                                                            |
+| **Memory (RAM)**          | 4GB                             | 8GB or more                                                                               |
+| **Processor (CPU)**       | 64-bit (x86_64) processor       | [x86-64-v2](https://en.wikipedia.org/wiki/X86-64#Microarchitecture_levels) 2GHz quad core |
+| **System firmware**       | BIOS / UEFI                     | UEFI                                                                                      |
+| **Display Adapter (GPU)** | Any onboard or dedicated adapter |                                                                                           |
+
+These are general recommendations. Your requirements may vary based on your intended use of the system.
+
+:::info
+
```
Comment (davidjharder) [inline docs/user/quick-start/installation/system-requirements.md:26]: The info box is a little vague. We should remove it or provide details on which cards we support, IMO

File: docs/user/quick-start/installation/system-requirements.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: System Requirements
+summary: Hardware Requirements Guide
+---
+
+# System Requirements
```
Comment (davidjharder) [inline docs/user/quick-start/installation/system-requirements.md:6]: Sentence case pls

**Review by davidjharder — CHANGES_REQUESTED** [review]
See inline

File: docs/user/quick-start/installation/system-requirements.md (hunk 1 above)
Comment (TraceyC77) [reply docs/user/quick-start/installation/system-requirements.md:26]: I can see this. I had inspiration from the [Fedora requirements](https://docs.fedoraproject.org/en-US/fedora/latest/release-notes/welcome/Hardware_Overview/) for the wording. I've added the list they have for GPUs that are old enough to probably not support acceleration. Is this better?

File: docs/user/quick-start/installation/system-requirements.md (hunk 2 above)
Comment (TraceyC77) [reply docs/user/quick-start/installation/system-requirements.md:6]: Fixed

File: docs/user/quick-start/installation/system-requirements.md (hunk 1 above)
Comment (davidjharder) [reply docs/user/quick-start/installation/system-requirements.md:26]: That's perfect

**Review by davidjharder — APPROVED** [review]

**Pull request merged**